### PR TITLE
XMLWriter::*Ns() accepts null in $prefix

### PR DESCRIPTION
**Pull request by spaze**
Hi,

> `prefix`: The namespace prefix. If `prefix` is **`NULL`**, the namespace will be omitted.

Docs:
https://www.php.net/xmlwriter-start-attribute-ns
https://www.php.net/xmlwriter-start-element-ns
https://www.php.net/xmlwriter-write-attribute-ns
https://www.php.net/xmlwriter-write-element-ns

Thanks!

### CODE REVIEW

**Comment by ondrejmirtes** [comment]
Thank you! :)

**Comment by spaze** [comment]
That was quick, thanks :-) Just realized that similar change needs to be applied to `functionMap_php80delta.php` (which I didn't notice before, sorry!) came here to make and... :-)

**Comment by ondrejmirtes** [comment]
It might not need to be, check out what phpstan.org does, there's more complicated logic. If the signature is correct in https://github.com/phpstan/php-8-stubs, there's a high chance the analysis will be already correct too :)

**Comment by spaze** [comment]
Aha, thanks. Currently, analyzing `startElementNs(null, ...)` fails on PHP 7.4 (which is how I found this) but doesn't on PHP 8. I guess different PHP 8 signatures would explain this, am I right?

And yes, https://github.com/phpstan/php-8-stubs/blob/main/stubs/ext/xmlwriter/xmlwriter_write_element_ns.php *et al.* seems correct with `?string $prefix`.

**Comment by ondrejmirtes** [comment]
Yes, my plan is to sync PHP 8 stubs with 7.4 functionMap (in an automated way) for things that actually haven't changed in PHP 8 :)

**Comment by spaze** [comment]
I see, thanks! Looking forward to a new release so I don't need to do some magic to make analysis pass on both PHP 7.4 & 8.0! Got a bit confused when I've ignored the error message on 7.x and then analyzing on PHP 8.0 said "no error" so thanks for showing me php-8-stubs. (Yeah, I know I could somehow use different config files but nah I'll happily wait :-)

File: resources/functionMap.php [hunk 1]
```diff
@@ -13110,19 +13110,19 @@
 'XMLWriter::startDTDElement' => ['bool', 'qualifiedname'=>'string'],
 'XMLWriter::startDTDEntity' => ['bool', 'name'=>'string', 'isparam'=>'bool'],
 'XMLWriter::startElement' => ['bool', 'name'=>'string'],
-'XMLWriter::startElementNS' => ['bool', 'prefix'=>'string', 'name'=>'string', 'uri'=>'string'],
+'XMLWriter::startElementNS' => ['bool', 'prefix|null'=>'string', 'name'=>'string', 'uri'=>'string'],
```
Comment (spaze) [inline resources/functionMap.php:13113]: The `null` should be in the `'string'` part. Sorry for that, remedied in #388

**Pull request merged**